src/CoreWCF.Primitives/src/CoreWCF/Channels/BufferedMessageData.cs

```diff
@@ -7,7 +7,7 @@
 
 namespace CoreWCF.Channels
 {
-    internal abstract class BufferedMessageData : IBufferedMessageData
+    public abstract class BufferedMessageData : IBufferedMessageData
     {
```

Comment:
Leave class internal. You'll need to make a copy in WebHttp. Keep the interface public. The Message class uses the IBufferedMessageData interface so that's all that needs to be exposed.

Reply:
This is addressed in 347fbf310d184e092377224a522c3021ef9770b4.